pkg/sfu/downtrack.go

```diff
@@ -361,14 +362,14 @@ func NewDownTrack(params DowntrackParams) (*DownTrack, error) {
 		upstreamCodecs:      codecs,
 		kind:                kind,
 		codec:               codecs[0].RTPCodecCapability,
+		clockRate:           codecs[0].ClockRate,
 		pacer:               params.Pacer,
 		maxLayerNotifierCh:  make(chan string, 1),
 		keyFrameRequesterCh: make(chan struct{}, 1),
 		createdAt:           time.Now().UnixNano(),
 	}
 	d.bindState.Store(bindStateUnbound)
 	d.params.Logger = params.Logger.WithValues(
-		"mime", codecs[0].MimeType,
```

Comment:
remove this field as it could change in simulcast-codec case, change bind log to info to log the mime type

Reply:
@boks1971  Since `mime` field is removed.